internal/impl/kube.go

```diff
@@ -268,16 +262,17 @@ func (r *replicaSet) buildListenerService(lis *ReplicaSetConfig_Listener) (*core
 			Kind:       "Service",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      globalLisName,
+			Name:      lisServiceName,
 			Namespace: r.namespace,
 			Labels: map[string]string{
 				"lisName": lis.Name,
+				"version": version,
```

Comment:
I think the version should always be `r.dep.Id[:8]`, even if the user specifies a service name?